bangazonapi/models/product.py

```diff
@@ -61,9 +62,13 @@ def average_rating(self):
         total_rating = 0
         for rating in ratings:
             total_rating += rating.rating
+        try:
+            avg = total_rating / len(ratings)
+            return avg
+        except:
```

Comment:
Can you add the name of the exception here? It's better to be explicit about which exception the code is catching